### PR TITLE
Swope scale intro

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: multicluster_engine/main.adoc [hunk 1]
```diff
@@ -24,8 +24,19 @@ include::credentials/credential_ansible.adoc[leveloffset=+3]
...
+include::cluster_lifecycle/create_ocp_aws.adoc[leveloffset=+3]
+include::cluster_lifecycle/create_azure.adoc[leveloffset=+3]
+include::cluster_lifecycle/create_google.adoc[leveloffset=+3]
+include::cluster_lifecycle/create_vm.adoc[leveloffset=+3]
+include::cluster_lifecycle/create_openstack.adoc[leveloffset=+3]
+include::cluster_lifecycle/create_virtualization.adoc[leveloffset=+3]
+include::cluster_lifecycle/create_cluster_on_prem.adoc[leveloffset=+3]
+include::cluster_lifecycle/create_hosted.adoc[leveloffset=+3]
 include::cluster_lifecycle/create_infra_env.adoc[leveloffset=+3]
 include::cluster_lifecycle/create_intro.adoc[leveloffset=+3]
```
Comment (swopebe) [inline multicluster_engine/main.adoc:41]: create intro is misplaced

File: multicluster_engine/main.adoc [hunk 2]
```diff
@@ -66,11 +77,11 @@ include::cluster_lifecycle/scale_cluster_pool.adoc[leveloffset=+4]
 include::cluster_lifecycle/cluster_pool_rel_img_update.adoc[leveloffset=+4]
 include::cluster_lifecycle/cluster_pool_destroy.adoc[leveloffset=+4]
 include::cluster_lifecycle/clusterclaims.adoc[leveloffset=+3]
```
Comment (swopebe) [inline multicluster_engine/main.adoc:79]: clusterpool_intro.adoc
other topics

(clusterpool)

**Comment by oafischer** [comment]
Is it OK if I delete this branch?

**Comment by swopebe** [comment]
deleted, yes

**Pull request closed without merging**